e2e-chatbot-app-next/.env.test.with-db

```diff
@@ -0,0 +1,15 @@
+# Test environment WITH DATABASE (persistent mode)
```

Comment:
Maybe rename this file to `.env.playwright` or something if it's only used in tests? We have `.env.example` already for testing with or without a DB right?